src/CoreLib/deploy.fs

```diff
@@ -217,10 +219,10 @@ type DeploymentSettings() =
     /// ReSerialization For Cache
     static member val internal DefaultCacheSerializationLimit = 1 with get, set
     /// ReSerialization For Cache
-    static member val internal DefaultIOMaxQueue = 10 with get, set
+    static member val internal DefaultIOMaxQueue = 100 with get, set
```

Comment:
I think it is fine to use this as memory is controlled by the shared stack.  Using higher number may in some cases increase delay on per message basis, but most of our current workload is not sensitive to this.